src/client/modules/my/playerRankingTable/playerRankingTable.html

```diff
@@ -4,20 +4,46 @@
 
     <thead>
       <tr class="slds-line-height_reset">
-        <th>Date</th>
-        <th>Name</th>
-        <th>Ranking</th>
-        <th>Victory Points</th>
+        <th>Player</th>
+        <th>Games Played</th>
+        <th>Percentage of Games Played</th>
+        <th>Average Points per Game</th>
+        <th>Total Points</th>
+        <th>First Place Finishes</th>
```

Comment:
In Matt's table, he had "First Place Count" instead of "First Place Finishes", so maybe rename this for 1st to 6th place just so it's a little clearer as to what it's keeping track of

Reply:
I'm okay with either wording